README.md

```diff
@@ -1,6 +1,12 @@
 # Opni Insights service
 
-* This repository builds the opni-insights image which sets up an HTTP server that will return the breakdown of normal, suspicious and anomolous log messages by pod, namespace and workload between a specified time interval provided in nanoseconds. This server can also return the overall breakdown of normal, suspicious and anomalous log messages within a specified time interval by specifying a Get request to the overall_insights endpoint. It can also return the suspicious and anomalous log messages within a specified time interval by submitting a Get request to the logs endpoint.
+* This repository builds the opni-insights image which sets up an HTTP server. It sets up endpoints for many different insights between any time interval.
+    * Breakdown of normal, suspicious and anomalous log message by pod through the pod endpoint
```

Comment:
we can probably summarize line 4,5,6,7 with `breakdown of normal, suspicious and anomalous log messages by pod/namespace/workload/control-plane-component`?

Reply:
Maybe, I just want it to be clear which endpoint the user should submit a Get request to.